### PR TITLE
Jai/TLI 10.1.h -- Fixed Graph Axes and Bias Values

### PR DESCRIPTION
**Pull request by Jai0212**
**Description:**  
This PR updates the graphs on both dashboards to display bias (calculated as `1 - accuracy`) instead of accuracy. The changes include updating the calculation logic, modifying axis labels, and verifying the accuracy of the updates.

**Tasks Completed:**  
1. Identified all graphs on both dashboards that display accuracy.  
2. Updated the logic to calculate bias by subtracting accuracy values from 1.  
3. Modified axis labels to indicate "Bias" instead of "Accuracy."  
4. Tested the updated graphs to ensure the values and labels are accurate.  

**Acceptance Criteria:**  
- All graphs previously showing accuracy now represent bias (calculated as `1 - accuracy`).  
- Axis labels are updated to read "Bias" instead of "Accuracy."  
- Values displayed on the axes match the expected bias calculations.  
- Changes are consistent across both dashboards.  
- Updates are verified through testing and reviewed before deployment.  

**Testing Notes:**  
All graphs have been thoroughly tested to ensure correctness of displayed values and labels. The updates maintain consistency across both dashboards.

### CODE REVIEW

File: backend/app/use_cases/generate.py [hunk 1]
```diff
@@ -24,5 +24,18 @@ def execute(
         if output is None:
             output = []
 
+        new_output = []
```
Comment (Navnoorsinghmahal) [inline backend/app/use_cases/generate.py:27]: Why we added this code?

File: backend/app/use_cases/generate.py [hunk 2]
```diff
@@ -24,5 +24,18 @@ def execute(
...
+        for i in output:
+            new_output.append(
+                {
+                    "feature1": i.get_feature1(),
+                    "feature2": i.get_feature2(),
+                    "accuracy": 1 - i.get_accuracy(),
```
Comment (Navnoorsinghmahal) [inline backend/app/use_cases/generate.py:33]: Why are we subtracting by 1 here

**Review by Navnoorsinghmahal — COMMENTED** [review]
Code reviewed and looks great!!

File: backend/app/use_cases/generate.py (hunk 2 above)
Comment (Jai0212) [reply backend/app/use_cases/generate.py:33]: Previously, the output returned accuracy, but now we need to modify it to return bias instead, so we need to subtract by 1

**Pull request merged**